types/zoomist/index.d.ts

```diff
@@ -0,0 +1,181 @@
+// Type definitions for zoomist 1.1
+// Project: https://github.com/cotton123236/zoomist
+// Definitions by: scriptSQD <https://github.com/scriptSQD>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+// Minimum TypeScript Version: 4.1
```

Comment:
I understand that was due to usage of template literal types

Reply:
Yup, that's it. This is quite essential, because otherwise some styling won't work (e.g. if you say height: '10rem').